apps/web/src/components/logo/logo.tsx

```diff
@@ -1,15 +1,21 @@
 'use client'
 
+import { authContext } from '@/context/auth'
 import { useLanguage } from '@/context/language'
 import Image from 'next/image'
 import Link from 'next/link'
+import { useContext } from 'react'
 
 type LogoProps = { size?: number }
 export const Logo = ({ size = 24 }: LogoProps) => {
   const { language } = useLanguage()
 
+  const { user } = useContext(authContext)
```

Comment:
authContext already export `useAuth`

```suggestion
  const { user } = useAuth()
```